### DISCUSSION TITLE
Any example directory for `lora_pti` ?

### DISCUSSION THREAD

**Discussion by yankeguo** [cloneofsimo/lora#216] [Q&A]
Didn't find any explanation and examples in the repository

**Comment by rishabhjain** [comment]
There's one in the README itself.

export MODEL_NAME="runwayml/stable-diffusion-v1-5"
export INSTANCE_DIR="./data/data_disney"
export OUTPUT_DIR="./exps/output_dsn"

lora_pti \
  --pretrained_model_name_or_path=$MODEL_NAME  \
  --instance_data_dir=$INSTANCE_DIR \
  --output_dir=$OUTPUT_DIR \
  --train_text_encoder \
  --resolution=512 \
  --train_batch_size=1 \
  --gradient_accumulation_steps=4 \
  --scale_lr \
  --learning_rate_unet=1e-4 \
  --learning_rate_text=1e-5 \
  --learning_rate_ti=5e-4 \
  --color_jitter \
  --lr_scheduler="linear" \
  --lr_warmup_steps=0 \
  --placeholder_tokens="<s1>|<s2>" \
  --use_template="style"\
  --save_steps=100 \
  --max_train_steps_ti=1000 \
  --max_train_steps_tuning=1000 \
  --perform_inversion=True \
  --clip_ti_decay \
  --weight_decay_ti=0.000 \
  --weight_decay_lora=0.001\
  --continue_inversion \
  --continue_inversion_lr=1e-4 \
  --device="cuda:0" \
  --lora_rank=1 \

**Reply by yankeguo** [reply]
I mean, how to prepare a data directory, does it work with SDWebUI output directory in `Train` -> `Process Images`, or it only accept image files with filename as 'caption'

**Reply by rishabhjain** [reply]
Not sure what's SDWebUI is, but $INSTANCE_DIR above should point to folder which has images which need to be trained.

**Reply by yankeguo** [reply]
thanks for explaining, I misunderstood how lora_pti works
